src/App.jsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+    />
+    <Sum
+      b={5}
+    />
```

Comment:
look in the [checklist](https://github.com/mate-academy/react_sum/blob/master/checklist.md) before making PR
<img width="1078" alt="image" src="https://github.com/mate-academy/react_sum/assets/98764641/87bfef9f-f082-42a1-8c97-1b441279ced4">